### ISSUE TITLE
Waiting for status to be reported

### ISSUE THREAD

**Issue by mbohlool** [travis-ci/travis-ci#7396]
Travis-ci checks does not update in https://github.com/kubernetes-incubator/client-python

I've even added [ci skip] but they still stock on "Waiting for status to be reported"

Sample PRs:
https://github.com/kubernetes-incubator/client-python/pull/143
https://github.com/kubernetes-incubator/client-python/pull/137

**Comment by BanzaiMan** [comment]
Both of these builds will not run because the commit message explicitly tell us to skip.

**Comment by mbohlool** [comment]
I know, but the problem is github status is not updated and we cannot merge until the status is updated. I've forced update 143 but a new PR has the same problem: https://github.com/kubernetes-incubator/client-python/pull/146

and this one is not skipping ci.

**Comment by mbohlool** [comment]
I forced merge those PRs but the problem still exists. I would appreciate any update on it. To track the problem I created a dummy PR: https://github.com/kubernetes-incubator/client-python/pull/149

**Comment by BanzaiMan** [comment]
Looks like the GitHub hook was disabled. I've re-enabled it now.

Unfortunately, we missed the webhook deliveries of the outstanding PRs; please push new commits to respective branches, or close and reopen the PRs to force webhook deliveries to run the builds. Thanks.

**Comment by mbohlool** [comment]
It is fixed now. Thanks.

**Comment by bennyn** [comment]
We are having issues with PRs from external developers like this one: https://github.com/wireapp/wire-webapp/pull/1133

Travis is stuck with "Waiting for status to be reported":

![Screenshot](https://cloud.githubusercontent.com/assets/469989/25480467/27c62350-2b49-11e7-938c-bfee653d31e1.PNG)

**Comment by mbohlool** [comment]
My experience is I got to that point when I use `[skip ci]` on my commits. I stopped doing that and never encountered this.

**Comment by bennyn** [comment]
Yeah, but it's expected that it skips CI if you write `[skip ci]` in your commit messages^^ But we didn't do that. We also didn't write [[ci skip]](https://docs.travis-ci.com/user/customizing-the-build/#Skipping-a-build).

**Comment by jaques-sam** [comment]
I have the same issue. The solution we use here is to push another commit to the branch :-/

**Comment by ckhicks** [comment]
Having the same issue - does the webhook setting seem to be the culprit for most of you?

**Comment by BanzaiMan** [comment]
see #10204, please.
